docs/integrations/amazon-aws/aws-privatelink.md

```diff
@@ -80,6 +80,12 @@ Sumo Logic exposes AWS PrivateLink endpoints to different [regions that depend
     <td>https://collectors.jp.sumologic.com</td>
     <td>https://open-collectors.jp.sumologic.com</td>
     <td>ap-northeast-1</td>
+  </tr>
```

Comment:
PL beta doc is addressed here already. https://github.com/SumoLogic/sumologic-documentation/pull/4497